Makefile

```diff
@@ -101,9 +101,11 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 
 deps: controller-gen ## Download all dependencies if necessary.
 
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0)
+CONTROLLER_GEN = $(GOBIN)/controller-gen
```

Comment:
You are changing the dir for the bin
We currently use ./bin for the binaries pinned to the versions we want so we don't mix these in the GOBIN dir